docs/source/guides/distributed_training.mdx

```diff
@@ -0,0 +1,199 @@
+
+To alleviate that, `optimum-neuron` supports parallelism features enabling your to harness the full power of your Trainium instance:
+
+  1. [ZeRO-1](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/frameworks/torch/torch-neuronx/tutorials/training/zero1_gpt2.html): It is a optimization of data-parallism which consists in sharding the optimizer state (which usually represents half of the memory needed on the device) over the data-parallel ranks.
+  2. [Tensor Parallelism](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/libraries/neuronx-distributed/tensor_parallelism_overview.html): It is a technique which consists in sharding each of your model parameters along a given dimension on multiple devices. The number of devices to shard your parameters on is called the `tensor_parallel_size`. 
+  3. [Pipeline Parallelism](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/libraries/neuronx-distributed/pipeline_parallelism_overview.html): **coming soon!**
+
+
+The good news is that is it possible to combine those techniques, and `optimum-neuron` makes it very easy!
+
+<Tip>
+All the [example scripts](https://github.com/huggingface/optimum-neuron/tree/main/examples) provided in the `optimum-neuron` repo have those features implemented via the [`NeuronTrainer`].
+</Tip>
+
+## How to enable ZeRO-1?
+
+Whether you use the [`NeuronTrainer`] or decide to have your own training script that uses the [`NeuronAccelerator`], it is very easy to enable the ZeRO-1 optimization.
+
+### Via the `NeuronTrainer`
+
+```python
+from optimum.neuron import NeuronTrainingArguments, NeuronTrainer
+
+# To enable ZeRO-1, set the `zero_1` argument to `True` in the training arguments.
+training_args = NeuronTrainingArguments(
+    ...,
+    zero_1=True,
+)
+
+trainer = NeuronTrainer(
+    model,
+    args=training_args,
+    train_dataset=train_dataset,
+    eval_dataset=eval_dataset,
+)
+
+trainer.train()
+```
+
+<Tip>
+Since the [example scripts](https://github.com/huggingface/optimum-neuron/tree/main/examples) use the [`NeuronTrainer`], you can enable ZeRO-1 when using them by add the `--zero_1` flag
+to your command line.
+
+For example:
+
+```bash
+torchrun --nproc_per_node=2 examples/language-modeling/run_clm.py \
+    --model_name_or_path TinyLlama/TinyLlama-1.1B-Chat-v0.6 \
+    --dataset_name wikitext \
+    --dataset_config_name wikitext-2-raw-v1 \
+    --do_train \
+    --per_device_train_batch_size 1 \
+    --block_size 1024 \
+    --bf16 \
+    --zero_1 \
+    --output_dir my_training/
+
+```
+</Tip>
+
+### Via the `NeuronAccelerator`
+
+There is a little bit more work to do when not using the `NeuronTrainer`:
+  
+  1. (Optional) Wrap the optimizer class to make it lazy. When ZeRO-1 is enabled the original optimizer is overridden to use a sharded version of it so it is possible to load original optimizer lazily so that the optimizer state is not materialized until it is actually sharded.
```

Comment:
```suggestion
  1. (Optional) Wrap the optimizer class to make it lazy. When ZeRO-1 is enabled the original optimizer is overridden to use a sharded version of it. Hence, it is possible to load the original optimizer lazily so that the optimizer state is not materialized until it is actually sharded.
```